hack/install.sh

```diff
@@ -79,7 +79,7 @@ if [ -z "$lsb_dist" ] && [ -r /etc/os-release ]; then
 	lsb_dist="$(. /etc/os-release && echo "$ID")"
 fi
 
-lsb_dist="$(echo "$lsb_dist" | tr '[:upper:]' '[:lower:]')"
+lsb_dist="$(echo "$lsb_dist" | tr '[:upper:]' '[:lower:]'|  cut -d' ' -f1)"
```

Comment:
So why not just put `'elementary os'` in the `case` below, especially so we don't accidentally break some other distribution later?
ie:
```bash
case "$lsb_dist" in
  # ...
  ubuntu|debian|linuxmint|'elementary os')
      # ...
```